src/views/ProductDescription/ProductDescription.js

```diff
@@ -135,20 +137,27 @@ export default class ProductDescription extends Component {
               This way, the issuer can provide a simple setup routine for the
               customer by generating a QR code, that the customer can scan with
               the mobile wallet.
```

Comment:
This is going to be obsolete once the white label apps are in place, right?

Reply:
Yes, I think so :)